### ISSUE TITLE
[BUG] Reformatted file contains misaligned tags

### ISSUE THREAD

**Issue by hrjakobsen** [Riverside-Healthcare/djLint#121]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - ubuntu 21.04
 - Python 3.9.5
 - djlint, version 0.5.7

## Issue

Misaligned tags in source are not aligned in formatted output.

## How To Reproduce
In all cases the file is formatted with the following command
```sh
djlint --check --profile django test.html
```

```html
<!DOCTYPE html>
<html>
    <head>
        <script>
            var a = "TEST";
            var b = a;
    </script>              <-------- This is misaligned
    </head>
    <body>
        test
    </body>
</html>
```

I would expect the following result:
```html
<!DOCTYPE html>
<html>
    <head>
        <script>
            var a = "TEST";
            var b = a;
        </script>
    </head>
    <body>
        test
    </body>
</html>
```
Potentially with a slightly different coding style, but the tags should align.

The actual reformatted file is:
```html
<!DOCTYPE html>
<html>
    <head>
        <script>
            var a = "TEST";
            var b = a;
        </script>
</head>
<body>
    test
</body>
</html>
```

Furthermore the reformatting is not stable. Invoking the tool again will result in:
```html
<!DOCTYPE html>
<html>
    <head>
        <script>
            var a = "TEST";
            var b = a;
            </script>
</head>
<body>
    test
</body>
</html>
```
and further invocations will increase the indentation of `</script>`. Maybe this second part is a separate issue.

**Comment by christopherpickering** [comment]
I'm doing an update that lets trailing tags on <style and <script to be indented. The rest I'm leaving as is for now. Potentially we might want to format the trailing tag on multi line comments... -->. Right now its left as is.
